wire-ios/Wire-iOS/Sources/UserInterface/Conversation/ConversationRootViewController.swift

```diff
@@ -74,10 +81,45 @@ final class ConversationRootViewController: UIViewController {
         fatalError("init(coder:) has not been implemented")
     }
 
+    // MARK: - Override methods
+
+    override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(animated)
+
+        shouldAnimateNetworkStatusView = true
+        navBarContainer.navigationBar.accessibilityElementsHidden = false
+        conversationViewController?.view.accessibilityElementsHidden = false
+    }
+
+    override func viewDidDisappear(_ animated: Bool) {
+        super.viewDidAppear(animated)
+
+        navBarContainer.navigationBar.accessibilityElementsHidden = true
+        conversationViewController?.view.accessibilityElementsHidden = true
+    }
+
+    private var child: UIViewController? {
+        return conversationViewController?.contentViewController
+    }
+
+    override var childForStatusBarStyle: UIViewController? {
+        return child
+    }
+
+    override var childForStatusBarHidden: UIViewController? {
+        return child
+    }
+
     func configure() {
         guard let conversationViewController = self.conversationViewController else {
             return
         }
+        navHeight = navBarContainer.view.heightAnchor.constraint(equalToConstant: 44)
```

Comment:
What is this magic number?

Reply:
coming from UINavigationBarContainer.swift